cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -486,6 +490,17 @@ func (oi *OVFImporter) Import() (*daisy.Workflow, error) {
 	return w, nil
 }
 
+func (oi *OVFImporter) handleTimeout(w *daisy.Workflow) {
+	timeout, err := time.ParseDuration(oi.params.Timeout)
+	if err != nil {
+		oi.Logger.Log(fmt.Sprintf("Error parsing timeout `%v`", oi.params.Timeout))
+		return
+	}
```

Comment:
It appears that the workflow may run indefinitely given a malformed timeout value. I can imagine a couple of other options; what was the motivation for picking this one?

Options:
1. Proceed without a timeout
2. Fail immediately and ask the user to fix the parameter
3. Fallback on a reasonable value

Reply:
Workflows will not run indefinitely as there are Daisy step workflows. The global timeout is in addition to Daisy step workflows. 

The alternative was to calculate step workflows in some way to achieve global timeout. Due to non-deterministic way of calculating these and the maintenance burden if workflows change, I've decided to introduce one global timeout.

Reply:
makes sense